fjord/analytics/templates/analytics/analyzer/search.html

```diff
@@ -145,6 +145,12 @@
 
 {% block content_middle %}
   <div class="col full">
+    <div class="block graph secret">
+      {# A graph will be dynamically inserted in these divs by Flot. #}
+      <div class="histogram" data-histogram="{{ histogram|to_json }}"></div>
+      <div class="legend"></div>
+      <div class="note">{{ _('Note: All dates and times are in Pacific Time.') }}</div>
+    </div>
```

Comment:
The frontpage dashboard used to have a graph on it. A while back, I changed the graph so it only shows up for people who are in the analyzers group.

The use case is that Matt and crew want to be able to do a search and see the graph as a first pass for figuring out whether something is a big deal and needs looking into or not. This helps them field requests from other people about the data.

I left it on the front page because I didn't want to spend the time to move it. In redoing all the Elasticsearch stuff, I decided to finally move it.